v2/srv.go

```diff
@@ -11,15 +12,32 @@ import (
 // The function's inputs are the same as those for net.LookupSRV
 // To use in the correct order:
 //
-// count, orderedSRV, err := OrderedSRV(service, proto, name)
-// i := 1
-// for  i <= count {
-//   srv := orderedSRV[i]
-//   // Do something such as dial this SRV. If fails move on the the next or break if it succeeds.
-//   i += 1
-// }
+//	count, orderedSRV, err := OrderedSRV(service, proto, name)
+//	i := 1
+//	for i <= count {
+//	  srv := orderedSRV[i]
+//	  // Do something such as dial this SRV. If fails move on the the next or break if it succeeds.
+//	  i += 1
+//	}
```

Comment:
This new indentation was automatically added by `gofmt`. Let me know if you'd like to revert it.